imports/plugins/core/layout/client/templates/layout/admin/admin.js

```diff
@@ -10,6 +10,7 @@ import ToolbarContainer from "/imports/plugins/core/dashboard/client/containers/
 import Toolbar from "/imports/plugins/core/dashboard/client/components/toolbar";
```

Comment:
Do we even need this file anymore? I think all of this has been moved to various react components.

Reply:
I think not, as i've stated in the getFooterComponent helper function. I'm going to delete this file.